src/List/ListItem.js

```diff
@@ -422,22 +422,24 @@ class ListItem extends React.Component {
      );
   }
 
-  createTextElement(styles, data, key) {
-    const isAnElement = React.isValidElement(data);
-    const mergedStyles = isAnElement ?
-      Object.assign({}, styles, data.props.style) : null;
-
-    return isAnElement ? (
-      React.cloneElement(data, {
+  createTextElement = (styles, data, key) => {
+    if (React.isValidElement(data)) {
+      let style = Object.assign({}, styles, data.props.style);
+      if (typeof data.type === 'string') { // if element is a native dom node
+        style = this.context.muiTheme.prepareStyles(style);
```

Comment:
Haha funny you should ask, I just asked @nathanmarks about that on gitter, we'll need custom but supposedly it's doable.